content/en/docs/instrumentation/net/shim.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: OpenTelemetry Tracing Shim
-linkTitle: OpenTelemetry API Shim
+linkTitle: Tracing Shim
```

Comment:
What is the plan regarding docs for metrics? Do you want to create a separate page like `Metrics Shim`?

Reply:
Is there an otel api -compatible shim like there is for tracing? If so then perhaps, although it might be more like an "opentelemetry API shim" doc instead

Reply:
I thought there, but it seems I was mistaken 😬 
Reference: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Api/Metrics
I am resolving this comment.